src/Ares.php

```diff
@@ -1,46 +1,85 @@
 <?php
+/**
+ * This file is part of Ares.
+ * Copyright (c) 2015 Václav Vrbka (http://aurielle.cz)
+ */
 
-namespace Aurielle\Ares;
+namespace Grifart\Ares;
 
 use Nette;
 
+
 /**
  * @author Milan Matějček <milan.matejcek@gmail.com>
- * @author Václav Vrbka <aurielle@aurielle.cz>
+ * @author Václav Vrbka <vaclav.vrbka@grifart.cz>
  */
 class Ares extends Nette\Object
 {
+	/** @var HttpDriver */
+	private $httpDriver;
 
-	/** @var IRequest */
-	private $request;
+	/** @var Cache */
+	private $cache;
 
-	public function __construct(IRequest $request = NULL)
+
+	public function __construct(HttpDriver $httpDriver, Cache $cache)

```

Comment:
Ona `Cache` je obálka nad `Nette\Caching\Cache`, ne přímo jeho instance: https://github.com/Aurielle/ares/blob/c2e66d74d0bbcf0819e2ceb4fb83c2fc1d97d550/src/Cache.php

Ale souhlasím, že to není ideální, zakrývám tím pak závislost a ta obálka v podstatě nic nového nepřidává…